.github/workflows/ruby.yml

```diff
@@ -0,0 +1,38 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: [2.3.8, 2.4.6, 2.5.4, 2.6.2, 2.7.2, '3.0.0']
+        gem-version: [2.7.9, 3.2.3]
+        exclude:
+          - ruby-version: 3.0.0
+            gem-version: 2.7.9
+        include:
+          - ruby-version: 2.2.10
+            gem-version: 2.7.9
```

Comment:
We cannot 

RubyGems 3.2 tests Ruby 2.5+:
https://github.com/rubygems/rubygems/blob/faa8cf6cccd2ae18eba7191618ff639cc0ddfd7c/.github/workflows/ubuntu-rubygems.yml

RubyGems 2.7 tests Ruby 1.8-2.5:
https://github.com/rubygems/rubygems/blob/c66a90aee091bbdf92188c9fdc3439eb2e1684d1/.travis.yml

cf. https://stdgems.org/rubygems/